Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page/http_group_suggestion_model.ts

```diff
@@ -0,0 +1,39 @@
+import * as Beam from 'beam';
+import * as Nexus from 'nexus';
+import { LocalGroupSuggestionModel } from './local_group_suggestion_model';
+import { GroupSuggestionModel } from './group_suggestion_model';
+
+export class HttpGroupSuggestionModel extends GroupSuggestionModel {
+
+  /** Constructs an HttpGroupSuggestionModel.
+   * @param serviceClients - The clients used to access the HTTP services.
+   */
+  constructor(account: Beam.DirectoryEntry,
+      serviceClients: Nexus.ServiceClients) {
+    super();
+    this.account = account;
+    this.serviceClients = serviceClients;
+    this._model = new LocalGroupSuggestionModel([]);
+  }
+
+  public isLoaded(): boolean {
+    return this._isLoaded;
+  }
+
+  public async load(): Promise<void> {
+    const groups = await 
```

Comment:
Only carry this out if the model has not yet been loaded. This is a general principle of models, once they are loaded, further calls to load them do nothing.